native/NativeSampleApps/test/RestExplorerTest/src/com/salesforce/samples/restexplorer/ServerPickerActivityTest.java

```diff
@@ -26,7 +26,6 @@
  */
 package com.salesforce.samples.restexplorer;
 
-import static androidx.test.InstrumentationRegistry.getInstrumentation;
```

Comment:
Every change in this class is either fixing a warning or changing `Thread.sleep` to `device.waitForIdle` (which speeds things up).